### ISSUE TITLE
Provide more information when downloading/installing client for iOS and Android

### ISSUE THREAD

**Issue by brbledsoe** [expo/expo-cli#985]
I'm trying to open a React Native project on an iOS Simulator through Xcode (Expo-Cli 3.0.10 - iOS 11.4 iPhone 8 - Xcode 10.3) and it gets hung up at:
```
Downloading the latest version of Expo client app

Installing Expo client on iOS simulator
```

I'm expecting the Simulator to open with the Expo app installed with my project running but no luck. 

Here's the diagnostics: 

 Expo CLI 3.0.10 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.2 - /usr/local/bin/node
      npm: 6.11.2 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^34.0.1 => 34.0.4
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz => 0.59.8
    npmGlobalPackages:
      expo-cli: 3.0.10

I have read similar issues to this one on your GitHub page but none have helped. I'm wondering if there are problems with the versions I am using of either Xcode or the iOS Simulator.

**Comment by TheInternet** [comment]
I am also having this issue.

**Comment by aarshaw** [comment]
Has anyone found a workaround for this?

**Comment by dashcraft** [comment]
Same issue, can we get some help with this?

**Comment by lukett89** [comment]
+1

**Comment by lukett89** [comment]
I had the same issue and I solved it by upgrading the version of expo-cli to 3.3.0.
Hope it will work for you too!

**Comment by brentvatne** [comment]
We should show updates on the download progress and if some time passes without progress we should show some information on how to work around this -- you can download the clients from the website at https://expo.io/tools

**Comment by brentvatne** [comment]
cc @byCedric this could be a good one to work on that would provide lots of value

**Comment by EvanBacon** [comment]
fixed in #1364
